### PR TITLE
PrimateLabs.Geekbench.6 version 6.0.0

### PR DESCRIPTION
**Pull request by infinitepower18**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96684)

### CODE REVIEW

**Comment by stephengillie** [comment]
Validation is failing because the installer launches the application, which is a non-interactive benchmark test. Running the installer in interactive mode shows a clicked-by-default checkbox to autorun the application:
image

It appears that running the installer in silent mode installs with the checkbox checked - is there an installer switch to uncheck it and prevent the application from automatically launching after install?

**Comment by infinitepower18** [comment]
@stephengillie /S seems to install it without launching the application

**Comment by infinitepower18** [comment]
I just checked, it is installing without launching automatically. I don't know why it's failing.

https://user-images.githubusercontent.com/44692189/218866473-3c4ed3c5-0344-41f6-8957-8fe3ec5df593.mp4

File: manifests/p/PrimateLabs/Geekbench/6/6.0.0/PrimateLabs.Geekbench.6.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: PrimateLabs.Geekbench.6
+PackageVersion: 6.0.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://cdn.geekbench.com/k9ea2vqm/Geekbench-6.0.0-WindowsSetup.exe
+  InstallerSha256: 642ACB59A87CF9FABEF43231202E9F795357856D07862A5649A580F150A21C4B
```
Comment (stephengillie) [inline manifests/p/PrimateLabs/Geekbench/6/6.0.0/PrimateLabs.Geekbench.6.installer.yaml:10]:
```suggestion
  InstallerSha256: 642ACB59A87CF9FABEF43231202E9F795357856D07862A5649A580F150A21C4B
  InstallerSwitches:
    Silent: /S
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**